docs/visual-basic/programming-guide/language-features/procedures/snippets/how-to-call-an-event-handler/Class1.vb

```diff
@@ -0,0 +1,26 @@
+    '****************************************************************************
+    ' Special Form class to test Snippet5 -- changed Sub catchClose from Public to Protected 9/29/2005 for FxCop
+    ' Changed Protected catchClose to Private 4/6/2006 for FxCop -- jcooke
+    Public Class SpecialForm : Inherits System.Windows.Forms.Form
+        '<Snippet5>
+        ' Place these procedures inside a Form class definition.
+        Private Sub catchClose(sender As Object, e As System.ComponentModel.CancelEventArgs)
+            ' Insert code to deal with impending closure of this form.
+        End Sub
+        Public Sub formOpened()
```

Comment:
Besides the bad naming convention here. Where actually this method gets called?
I don't see it's called anywhere, nor tied with an event.

Reply:
This is event handler is registered on line 11 `AddHandler Me.FormClosing, AddressOf catchClose` to the `FormClosing` event.

Reply:
@IEvangelist Yes I see that it registers catchClose to the FormClosing event. But the method `formOpened` itself isn't called anywhere. I'd expect to see the `formOpened` method to be attached to Form.Load event or something.

Reply:
I'm sorry, I thought you were saying `catchClose`. Yeah, I see that now. It was probably registered on the form designer - and over time it may have been removed or something. This could probably be replaced with the .ctor instead.